src/atomic-components/index.ts

```diff
@@ -0,0 +1,2 @@
+export { HeadingTextComponent } from "./headingText.js";
+export type { HeadingTextProps } from "./headingText.js";
```

Comment:
If you add `"./components: "./dist/atomic-components"` to package.json, does `import {HeadingTextComponent} from "@yext/visual-editor/atomic-components"` work in the starter?

Reply:
It should but that wouldn't be the path we want. I think all atomic and macro should just be at /components. The editor ones can still live at root.